gradle.properties

```
@@ -12,7 +12,7 @@ ballerinaGradlePluginVersion=1.0.0
 gsonVersion=2.8.8
 
 stdlibIoVersion=1.4.0
-stdlibHttpVersion=2.7.0
+stdlibHttpVersion=2.7.3-20231012-153400-1cc836c
```

Update http version after release
```suggestion
stdlibHttpVersion=2.7.3-20231012-153400-1cc836c
```